python/ray/serve/api.py

```diff
@@ -617,6 +626,40 @@ def connect() -> Client:
     return Client(controller, controller_name, detached=True)
 
 
+def get_current_backend_tag() -> BackendTag:
+    """When called from within a backend, return its backend tag.
+
+    Raises:
+        RayServeException if not called from within a Ray Serve backend.
+    """
+    if _INTERNAL_REPLICA_CONTEXT is None:
+        raise RayServeException("Called `serve.get_current_backend_tag()`"
+                                "from a function that is not running in a "
+                                "Ray Serve backend on this cluster, so it "
+                                "has no backend tag yet. Please "
+                                "use client.create_backend() to run your "
+                                "backend function using Ray Serve.")
```

Comment:
I think we could make this more concise & clearer by just saying `serve.get_current_backend_tag() may only be called from within a Ray Serve backend.`

Reply:
(same for replica tag)